argo-workflows/nautobot-interface-sync/docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+
+The purpose of the workflow templates in this folder is to create or update device interfaces
+in Nautobot from Redfish info that we get from a device using OBM credentials.
+
+The sensor listens for Nautobot interface updates and triggers `get-obm-creds` to get OBM credentials, which
+is a dependency for workflows in this folder. Examples of dependencies can be found in a deps folder.
+As the next step, the sensor triggers our `synchronize-interfaces-to-nautobot` workflow, which will get the Redfish
+info from a server and will create or update device interfaces in the Nautobot.
+
+## Servers/OBMs supported
+The code is using Sushy library to get the Redfish info, but to support older Redfish versions, there are
+workarounds created in the code.
```

Comment:
```suggestion
The code utilizes the Sushy library to obtain Redfish information. However, to accommodate older versions of Redfish, several workarounds have been implemented within the code.
```

Consider explaining what the workarounds are - I assume that you meant fetching info from the proprietary HP JSON endpoint?

Reply:
not too sure if explaining exactly the workarounds will help as there a quite a few